setup.py

```diff
@@ -25,7 +25,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/engineerjoe440/pycev",
-    packages=['pycev'],
+    packages=["pycev"],
```

Comment:
Drat!

I was mistaken in my memory, and it looks like this needs to change as follows:

```diff
    url="https://github.com/engineerjoe440/pycev",
-   packages=["pycev"],
+   py_modules=["pycev"],
    classifiers=[
```

See [example here](https://gist.github.com/arne-cl/c7d156b48a8ecf27f207) for reference.

Reply:
Got it